Saga + Class fixes for "M15 Saga" and "Mainframe DFC"

VerumCH opened the pull request:
Made a number of fixes to Sagas and Classes for both the standalone `magic-m15-saga.mse-style` as well as the general DFC style, `magic-m15-mainframe-dfc.mse-style`.

Details also included in commit descriptions.

### Summary

* Fixed card images so that the text boxes have the correct color background.
* *(Not noted in commit descriptions)* Slightly fixed bottom edge of Saga/Class frames to have a complete black border around the image/text frame.
* Replaced the "paper" logic by adding additional card images specific to classes in `class/`, to ensure class text boxes also have the correct background and that text boxes work properly with blending.
* Fixed card images to add the correct DFC title cards with cutouts for the DFC icon. *(DFC only)*
* Added `...card2.jpg` images for both Sagas and Classes so that the back-side cards have the correct darkened title and typeline cards. *(DFC only)*
* Fixed `level.png` by rebuilding it from scratch with transparency built in, using high-res scans of real cards for reference.
* Fixed the `fullmask` mask files used for "nyx" overlay to include the bottom section of the card below the typeline.
* Fixed `anyx.png` to use `cnyx.png` with some minor color corrections, using high-res scan of an actual `Artifact Enchantment - Saga` as reference.
* Updated `amark.png` to be more in line with actual existing Artifact Sagas. (Well, the one.) Substantially lightened and minorly adjusted the hue of the center of the banner to closely match the artifact text box background, and updated the very top of the banner to be gold instead of black.
* Added "Inverted Triangle" and `specialize_down.png` for the DFC corner icon of the downward-pointing triangle that is used on the back side of non-Day-Night TDFCs. *(DFC only)*
* Added/updated necessary blend mask images and made associated necessary changes to `style`.

### Card Samples

![Saga Sample](https://github.com/MagicSetEditorPacks/Full-Magic-Pack/assets/7446254/615434e1-5334-4ff1-b1bc-50c82ef2dd59)
![Class Sample M](https://github.com/MagicSetEditorPacks/Full-Magic-Pack/assets/7446254/ede90f35-267c-4683-89a0-a55cc05b0bac)
![Front Saga](https://github.com/MagicSetEditorPacks/Full-Magic-Pack/assets/7446254/d54c1440-c711-4460-83d7-fe80e9ad11bf)
![DFC Saga M](https://github.com/MagicSetEditorPacks/Full-Magic-Pack/assets/7446254/600d6912-c000-4607-b8cf-62ad7b63be38)
![Artifact Saga Sample](https://github.com/MagicSetEditorPacks/Full-Magic-Pack/assets/7446254/1c83c61c-0bfe-4383-afc2-54339c51d846)
![DFC Artifact Saga](https://github.com/MagicSetEditorPacks/Full-Magic-Pack/assets/7446254/c86cdc8b-0511-40b4-a9e4-714f70a1d645)

### Card References

![cmm-867-the-eldest-reborn](https://github.com/MagicSetEditorPacks/Full-Magic-Pack/assets/7446254/705d7dec-f064-4594-91e2-625a9343770a)
![afr-125-warlock-class](https://github.com/MagicSetEditorPacks/Full-Magic-Pack/assets/7446254/8036bbe3-df4d-40d2-828a-6bfcebdbb2ac)
![clb-846-firja-s-retribution](https://github.com/MagicSetEditorPacks/Full-Magic-Pack/assets/7446254/47e0ca7d-b4fb-41cf-b7f4-09d5a88cc472)
![neo-124-tribute-to-horobi](https://github.com/MagicSetEditorPacks/Full-Magic-Pack/assets/7446254/5efb451c-8f7e-4950-8218-004fe7be03ee)
![mom-125-the-true-scriptures](https://github.com/MagicSetEditorPacks/Full-Magic-Pack/assets/7446254/91510cd3-2f29-4a6c-8672-268e0805254e)
![unf-189-greatest-show-in-the-multiverse](https://github.com/MagicSetEditorPacks/Full-Magic-Pack/assets/7446254/daa09cb5-90ea-4242-b621-41a2ca589440)

### Known Visual Issues Remaining

*(For Sagas and/or Classes, specifically. I may have forgotten one or two other minor things.)*

* Text boxes for back side of DFCs should be darker than front side.
* Front-side Sagas that flip into Creatures should have a notch at the bottom left and a P/T reminder. (See Tribute to Horobi above.)
* Back-side Sagas should have the DFC icon in the upper *right* corner, instead of the left. (See The True Scriptures above.)
* All Saga banners should now have the top section of the banner, which appears to wrap around behind the frame of the card, in gold rather than using the same color as the banner. (See all above Saga reference cards *except for* The Eldest Reborn, which is the oldest of the listed cards.)
* Saga chapter dividing lines should actually taper away at the horizontal edges and have a faint fade effect above the line, rather than just being a flat horizontal line all the way across. (See all above Saga reference cards.)

### Other Notes

I didn't check around any other Style templates to see if these same changes should be ported elsewhere. The Mainframe DFC and M15 Saga Styles seem to be the primary intended templates for DFC/non-DFC Sagas and Classes, respectively.

CajunAvenger commented:
rad. will see how much of the todo stuff i can tackle this week on top of code review. probably merge without if it doesn't look promising

VerumCH commented:
of note, the notch in the bottom left for the saga creatures seems to also come with a different, shorter chapter banner (by just a couple mm). That one is probably the most work, but all the assets are probably already around somewhere aside from the cropped banner.

I believe I know where to find the correct title card for the back side sagas with the icon in the right corner, I might take a look at that tomorrow to see if I can splice the images together nicely

the darker text box backgrounds technically would apply to all backside cards, not just sagas. not sure if there are raw images of that correct-color background texture kicking around anywhere, would be a fairly simple fix if so

VerumCH commented:
Added a couple commits to fix up a few of the issues I mentioned prior:

* Saga chapter dividers
* Back side of DFC text box color
* Back side Saga DFC icon in top right
* *(Not mentioned in commit details)* Also includes some small fixes to title and type box background colors for Sagas and Classes for a couple colors that were a bit off.

### Card Samples

![Front Saga](https://github.com/MagicSetEditorPacks/Full-Magic-Pack/assets/7446254/1fe9d030-ce19-4a94-b317-7f93f92bdca4)
![Front Creature](https://github.com/MagicSetEditorPacks/Full-Magic-Pack/assets/7446254/08bbbfc5-5d08-4e5a-b0e9-ea73a506cd06)
![MDFC Creature](https://github.com/MagicSetEditorPacks/Full-Magic-Pack/assets/7446254/dad4b1af-7c8a-453b-b223-835d8eecfab2)

VerumCH commented:
Added another couple minor fixes, sorry for the churn.

* Corrected calculation for `chapter0` in "M15 Saga" style to make the first divider line and start of the first chapter text box land at the correct spot, in line with the top right "corner" of the saga banner. (145px `->` 155px) *(Mainframe DFC sagas already used this value.)*
* Minor tweaks to `chapter-line.png` divider to more closely match the more subtle appearance on real Saga cards.
* Minor tweak to `level.png` divider to include the faint beveled edge on the top side of the black divider line.

### Card Samples

![Saga Sample](https://github.com/MagicSetEditorPacks/Full-Magic-Pack/assets/7446254/65f6330b-1bf7-4295-ac24-7c986f66689a)
![Class Sample M](https://github.com/MagicSetEditorPacks/Full-Magic-Pack/assets/7446254/33de1b73-671d-47c2-8cb2-4e873be7e61b)
